src/parser/priest/holy/modules/features/HolyPriestHealingEfficiencyBreakdown.tsx

```diff
@@ -83,8 +83,7 @@ class HolyPriestHealingEfficiencyBreakdown extends HealingEfficiencyBreakdown {
                 </tr>
               </thead>
               <tbody>
-                // @ts-ignore
-                <this.HealingEfficiencyTable tracker={tracker} showHealing={this.state.showHealing} />
+                {this.HealingEfficiencyTable({ tracker, showHealing: this.state.showHealing })}
               </tbody>
```

Comment:
There are some more ts-ignore in this file, are these removable with this change as well?

Reply:
I have no idea. I did the minimal change to make it compile and no more.

Reply:
All good!